build-logic/src/main/kotlin/com.hedera.hashgraph.aggregate-reports.gradle.kts

```diff
@@ -17,24 +14,23 @@ import java.io.BufferedOutputStream
  * limitations under the License.
  */
 
+import java.io.BufferedOutputStream
+import net.swiftzer.semver.SemVer
+import org.owasp.dependencycheck.reporting.ReportGenerator
+
 plugins {
     id("org.sonarqube")
+    id("org.owasp.dependencycheck")
     id("lazy.zoo.gradle.git-data-plugin")
 }
 
-//  Configure the Sonarqube extension for SonarCloud reporting. These properties should not be changed so no need to
+// Configure the Sonarqube extension for SonarCloud reporting. These properties should not be changed so no need to
 // have them in the gradle.properties defintions.
 sonarqube {
     properties {
         property("sonar.host.url", "https://sonarcloud.io")
         property("sonar.organization", "hashgraph")
-        property("sonar.projectKey", "com.hedera.hashgraph:hedera-services")
-        property("sonar.projectName", "Hedera Services")
```

Comment:
Moved to the root 'build.gradle.kts' files to be different in the two (hedera-node vs platform-sdk) builds

Reply:
@hendrikebbers Based on the size and impact of this PR and due to the remaining namespace/publish issues we need to address, this is the least of my concerns. Let's leave it for now and open an issue to resolve later as a separate PR once the critical issues are sorted.

Reply:
Opened issue https://github.com/hashgraph/hedera-services/issues/8091